mixer/adapter/stdio/stdio.go

```diff
@@ -139,9 +131,25 @@ func GetInfo() adapter.Info {
 			metric.TemplateName,
 		},
 		DefaultConfig: &config.Params{
-			LogStream:    config.STDOUT,
-			MetricLevel:  config.INFO,
-			OutputAsJson: false,
+			LogStream:          config.STDOUT,
+			MetricLevel:        config.INFO,
+			OutputLevel:        config.INFO,
+			OutputAsJson:       true,
+			RotationMaxAge:     30,
```

Comment:
Shall we rename these to MaxRotationDays. Trying to embed the format in the anme

Reply:
Or RotationDaysMax